trino-connector/src/test/java/com/datastrato/gravitino/trino/connector/catalog/hive/TestHiveCatalogPropertyConverter.java

```diff
@@ -28,4 +30,13 @@ public void testConverter() {
     Assert.assertEquals(re.get("hive.compression-codec"), "ZSTD");
     Assert.assertEquals(re.get("hive.unknown-key"), null);
   }
+
+  @Test
+  // To test whether we can load property metadata from HiveTablePropertiesMetadata successfully.
+  public void testPropertyMetadata() {
+    for (Map.Entry<String, PropertyEntry<?>> entryEntry :
+        new HiveTablePropertiesMetadata().propertyEntries().entrySet()) {
+      System.out.println(entryEntry.getKey() + " " + entryEntry.getValue());
```

Comment:
Why do we need this test? if you want to have a test, you should make it more formal. Also never use system output in the codes. As I remembered, it will fail the style check, but seems not.

Reply:
I want to test whether we can load `bundled-catalog` jar succfully, so, I use `Logger` instead?

Reply:
I think you should use junit api to verify, otherwise it is not a UT, IMO.

Reply:
OK, I will change it accordingly.